src/ripple/app/misc/NetworkOPs.cpp

```diff
@@ -2079,6 +2100,19 @@ Json::Value NetworkOPsImp::getServerInfo (bool human, bool admin)
                 info[jss::load_factor_cluster] =
                     static_cast<double> (fee) / base;
         }
+        if (escalationMetrics)
+        {
+            if (escalationMetrics->expFeeLevel !=
+                    escalationMetrics->referenceFeeLevel)
+                info[jss::load_factor_fee_escalation] =
+                    static_cast<double> (escalationMetrics->expFeeLevel) /
+                        escalationMetrics->referenceFeeLevel;
```

Comment:
This will be a problem with the Json class as it (Nor Javascript) supports the precision. Take for instance the following example:

    Json::Value jv = Json::Value(Json::objectValue);
    jv[jss::load_factor_fee_escalation] = static_cast<double>(3 / 254);
    auto d = jv[jss::load_factor_fee_escalation].asDouble();
    auto s = jv.toStyledString();

3/254 = 0.0118110236220472 yet 'd' is 0.0 and the value of 'load_factor_fee_escalation' in 's' will incorrectly be 0.